utils/inflectors.ts

```diff
@@ -9,6 +9,13 @@ export function titleize(str:string) {
     });
 }
 
+// Take a title string and replace all spaces with commas, convert to lowercase
+
+export function urlize(str:string): string {
+
+  return str.replace(/\ /g, '-').toLowerCase();
+}
+
```

Comment:
This is actually a thing called `slugify`, as you are making URL 'slugs'!  Not that we need them, but there are whole packages dedicated to handling this kind of thing in multiple languages, etc.